src/main/java/gift/order/MessageTemplate.java

```diff
@@ -0,0 +1,7 @@
+package gift.order;
+
+public record MessageTemplate(
+    String object_type,
```

Comment:
자바 컨벤션은 가급적 지켜주는 것이 좋을 것 같아요ㅎㅎ
```suggestion
    String objectType,
```